src/buffer_logger/can_buffer_logger.py

```diff
@@ -44,13 +42,16 @@ async def listen_to_network(self):
         # Uses circular buffer implementation which overwrites
         # oldest index with new data once the buffer is full.
         index = 0
-        while True:
-            can_msg = await self.reader.get_message()
-            # Sleeps 0.1 seconds so doesn't try to read all messages at once.
-            await asyncio.sleep(0.1)
-            self.buffered_data[index] = can_msg
-            index = (index + 1) % self.can_logging_buffer
-        self._closing_async_listener()
+        try:
```

Comment:
moet de try/except niet binnen de while loop?

Reply:
Over het algemeen maakt het niet veel uit. https://www.quora.com/Should-exception-handling-go-inside-or-outside-a-loop 
Maar bij onze situatie wel, stel je hebt de exception. dan stopt het met de while loop in de huidige situatie.
Als try/except binnen de loop zit dan gaat het toch door itereren en dat is niet gewenst.